pkg/utils/constant.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+const (
+	PortNo  = ":8080"
+	HMDB    = "HMDB"
+	Doctors = "Doctors"
+)
+
+type Doctor struct {
```

Comment:
Why this is here? Also why inside constant file?

Reply:
than where it should be ?